scripts/post_leap.sh

```diff
@@ -35,4 +35,11 @@ if [[ ! -f "${UPGRADE_LEAP_MARKER_FOLDER}/deploy-rpc.complete" ]]; then
 else
   log "deploy-rpc" "skipped"
 fi
+
+if [ "$UPGRADE_ELASTICSEARCH"=="yes" ]; then
+    pushd /opt/rpc-upgrades/scripts
+        source ./elasticsearch_upgrade.sh
```

Comment:
Just ruin this script in a screen/tmux and wrap some logic around a lock file, similar to https://github.com/rsoprivatecloud/openstack-ops/blob/master/playbooks/update-hp-firmware.yml#L43-L57 so we can background the task

Reply:
What about adding 'fire and forget' to this task? So main leap can finish, then user could check reindex by polling es status.
Like
- name: reindex
   command: xxx
   async: 45
   poll: 0

Reply:
IMO, based on current script, when leap progress reaches ES upagrade part, whole leaping has arrived the very end, so there's no other upgrades wait to do. And also user can simply switch out tmux session then polling ES status to check reindexing progress. We made this guide into document too.

Reply:
I still prefer to background the task and especially we need to increase the ansible command timeout, can't remember which setting this was. Otherwise the thread gets interrupted prematurely. Hence I prefer to nohup the task no matter if it runs in a tmux and at the end of a script